registry.json

```diff
@@ -1,4 +1,15 @@
 [
+{
+    "ticker": "AMDLO",                                                       
+    "poolId": "2730ca79a60b3b3f29b709bd39c21d66cb2c83e5441c479f755d0a3d", 
+    "social": {       
+      "website": "https://armadillo-pool.org",
+      "twitter": "Armadillo_Pool", 
+      "instagram": "",
```

Comment:
Please add the "facebook" and "youtube" keys, even if they are blank

Reply:
Rectified now. thanks.